ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -161,6 +161,25 @@ conformance_test(
     ],
 )
 
+conformance_test(
+    name = "conformance-test-multi-participant-pruning",
+    ports = [
+        6865,
+        6866,
+    ],
+    server = ":app",
+    server_args = [
+        "--contract-id-seeding=testing-weak",
+        "--index-append-only-schema-unsafe",
+        "--participant=participant-id=example1,port=6865",
+        "--participant=participant-id=example2,port=6866",
+    ],
+    test_tool_args = [
+        "--verbose",
+        "--include=MultiParticipantDivulgencePruningIT",
+    ],
+)
+
```

Comment:
Looking at all these conformance tests, they it is not clear to me that we:
1. We run all the relevant combinations, and avoid missing test coverage for specific ways to run the participant.
2. We have minimized the set of conformance tests being run; both to minimize CI time, as well, as to make it clear how to extend it.

Here concretely, I'd expect that we run multi-participant conformance tests for both DB schemas. 

For a cleanup, what I'd expect is that we add commentary to the build file that gives a rationale as to what tests are required and why, incl. how the set of tests is to be extended.

Reply:
Good observations. Extracted in separate story.